test/integration/serverless/parse-json.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+var readline = require('readline');
+var rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+  terminal: false
+});
+
+rl.on('line', function(line){
+    try {
+      const obj = JSON.parse(line)
```

Comment:
Just for my understanding, we can't use JQ for pretty printing, because some of the lines aren't valid JSON?

Reply:
exact, catching error would be easier with JSON.parse